plenum/server/consensus/view_change_service.py

```diff
@@ -283,7 +283,7 @@ def _finish_view_change_if_needed(self):
     def _finish_view_change(self):
         # Update shared data
         self._data.waiting_for_new_view = False
-        self._data.prev_view_prepare_cert = self._new_view.batches[-1].pp_seq_no if self._new_view.batches else None
+        self._data.prev_view_prepare_cert = self._new_view.batches[-1].pp_seq_no if self._new_view.batches else 0
```

Comment:
Please note that ppSeqNo starts with 1, so 0 here can be considered as None or don't know (agree that we need a comment about this).
The assumption 'don't know == 0' is more safe from the code point of view since we are doing (and may do more in future) comparisons (>=, <=), so we don't have to check for None every time. 
There is another non-obvious benefit of setting it to 0 and not last_ordered: we do allow re-ordering for everything in between low watermark and prepared certificate. So setting it to last_ordered allows to  potentially flood the pool with unnecessary re-ordering.